src/components/SelectNetworkAndAccount.svelte

```diff
@@ -66,13 +66,13 @@
 
   async function networkChanged() {
     isLoading = true;
+    isCustomNetwork = selectedNetwork?.id === NetworkType.CUSTOM;
```

Comment:
By moving this before the await, it removes the lag of the custom field appearing and disappearing